demo/simple.html

```diff
@@ -9,7 +9,7 @@
         <script src="../jquery.orgchart.js"></script>
         <script>
         $(function() {
-            $("#organisation").orgChart({container: $("#main")});
+            $("#organisation").orgChart({container: $("#main") , direction:'ltr'});
```

Comment:
yes,it's true,
read me updated about demo
link to live demo can be on your site :)